Mathlib/CategoryTheory/Sites/Whiskering.lean

```diff
@@ -114,9 +149,12 @@ def mapMultifork :
 
 end GrothendieckTopology.Cover
 
+section Multicospan
+
 variable [∀ (X : C) (S : J.Cover X) (P : Cᵒᵖ ⥤ A), PreservesLimit (S.index P).multicospan F]
 variable [∀ (X : C) (S : J.Cover X) (P : Cᵒᵖ ⥤ A), PreservesLimit (S.index P).multicospan G]
 variable [∀ (X : C) (S : J.Cover X) (P : Cᵒᵖ ⥤ A), PreservesLimit (S.index P).multicospan H]
+variable (F)
 
 theorem Presheaf.IsSheaf.comp {P : Cᵒᵖ ⥤ A} (hP : Presheaf.IsSheaf J P) :
```

Comment:
We no longer require this a separate statement, the proof should be moved to the `instance` declaration below.